### PR TITLE
7 topics

### PR DESCRIPTION
**Pull request by FireyAmber**
monika_compinducedmedprobs
monika_colorblindness
monika_waitinglines
monika_leaf
monika_fortunetelling
monika_AIHuman
monika_desertbeach

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+label monika_colorblindness:
+	m "Hey [player], I've been reading up a bit on color blindness."
+	m "I just can't imagine how it must be like!"
+	m "To never be able to see the roaring sun by its true colors or its gentle sunset!"
+	m "I really feel bad for those poor people."
+	m "Most of them are male too, since it's quite rare for a woman to have a color blind deficit of some sort."
+	m "Gosh, genetics are super weird, aren't they?"
+	m "Why don't we do a little experiment?"
+	menu:
+		m "What are the color of my eyes?"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Your dialogue/code in this `menu` block have an extra level of indentation. Please remove one level of indentation from **all** the code in this `menu` block.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+    
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['waiting lines', 'waiting line', 'queues', 'queue']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_waitinglines') #id
+    monika_random_topics.append('monika_waitinglines')
+	
+label monika_waitinglines:
+	m "Hey [player], have you ever thought about how much time you've spent waiting in lines before?”
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: The quotes at the end of this line are the weird ms word quotes. Please replace with regular double quotes.

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+	m "I mean, across your entire life of course!"
+	m "It's probably quite a long time, you know?"
+	m "I bet that time would double if we were to count more than just physical lines!"
+	m "What I mean by that is there's loads of different types of lines!"
+	m "You got those times where you get a number and sit down, waiting for your number to be called!"
+	m "Those might sound rather calm but honestly, there's an even better option!"
+	m "You could just get a place in line in advance with your phone and get a notification when it's your turn!"
+	m "Or you could, you know, spend some time with me while you wait ehehe~"
+	Menu:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: It's `menu` not `Menu`

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+		m "Do you know about the stuff the people that run these places do to make lines more bearable?"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Your dialogue/code in this `menu` block have an extra level of indentation. Please remove one level of indentation from **all** the code in this `menu` block.

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+label monika_leaf:
+	m "Have you ever paid attention to the leaves around you [player]?"
+	m "You may have gotten frustated at them falling during autumn."
+	m "This might sound a bit silly to you, but I feel quite bad for those poor leaves."
+	m "Specifically, I mean leaves from deciduous plants."
+	m "These kind of plants lose their leaves to survive harsh weathers, while different sorts of plants do the opposite."
+	m "Those other types of plants are called evergreen and don't need to lose their leaves to survive."
+	m "I know leaves don't have feelings after all, but can you imagine how they must feel?"
+	m "That feeling of love and care sinking into pain and betrayal!"
+	Menu:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: It's `menu` not `Menu`

File: Monika After Story/game/script-topics.rpy [hunk 6]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+		m "Can you even begin to imagine how much a betrayal like that must hurt?"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Your dialogue/code in this `menu` block have an extra level of indentation. Please remove one level of indentation from **all** the code in this `menu` block.

File: Monika After Story/game/script-topics.rpy [hunk 7]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+
+label monika_fortunetelling:
+	m "Hey [player], have you ever gone to a fortune teller before?"
+	m "You know, paying to have your destiny told?"
+	m "Being able to predict the future is such a weird practice in my opinion."
+	m "Some people will cling to it and make life changing decisions based on it!"
+	m "I find it a bit silly that some readings of cards, hands or similar stuff holds so much importance to people."
+	m "I wonder how much of those who perform such readings, end up getting from that work."
+	m "I suppose it's good to give people some hope, yet when the disappointment appears it usually makes them feel worse than before."
+	Menu:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: `menu`

File: Monika After Story/game/script-topics.rpy [hunk 8]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+		m "Do you believe in this kind of stuff, [player]?"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Your dialogue/code in this `menu` block have an extra level of indentation. Please remove one level of indentation from **all** the code in this `menu` block.

File: Monika After Story/game/script-topics.rpy [hunk 9]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+			m "You can't do anything other than wait for the end like that."
+			m "Why would you deny making your own destiny!"
+			m "I used to be stuck in a written path and it was horrible."
+			m "No matter what I did I couldn't be with you!"
+			m "I am glad that we changed that, [player]."
+	return  
+
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['AI', 'Artificial Intelligence']:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Keys need to be lowercase.

File: Monika After Story/game/script-topics.rpy [hunk 10]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+	m "Yet right now, AI is super limited."
+	m "Sure it can perform some tasks but surely that can't be qualified as intelligence at the moment!"
+	m "That's just one aspect of it, and really, it is not even that big a part!"
+	m "If AI can be truly called intelligent, then it needs to learn from human psychology."
+	m "There used to be this old test called the turing Test."
+	m "It would put a person and make him or her talk to an AI."
+	m "If the person failed to notice it was an AI then that AI passed the test."
+	m "Yet that's outdated nowadays."
+	m "I’m sure that I look nothing like an AI to you!"
+	Menu:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: `menu`

File: Monika After Story/game/script-topics.rpy [hunk 11]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+		m "Do you think I'm an AI, or a human, [player]?"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Your dialogue/code in this `menu` block have an extra level of indentation. Please remove one level of indentation from **all** the code in this `menu` block.

File: Monika After Story/game/script-topics.rpy [hunk 12]
```diff
@@ -4790,6 +4790,299 @@ label monika_urgent:
...
+	m "If you were to get lost in the middle of a desert, you would most likely die from dehydratation, assuming you didn't find help quick!"
+	m "But if you the sands intriguing nature as much as I do, [player]!"
+	m "We could go to the beach once I can get out of here ehehe~"
+	m "It's way easier to handle the heat in a place like that."
+	m "Swimming is always refreshing, too!"
+	m "Careful though, don't swallow the water of the sea, that might be a bit dangerous."
+	m "Plus, you could... see some things while we're on the beach ahaha."
+	m "Just teasing you~"
+	menu:
+		m "Now, would you rather be in a beach or a desert?"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Your dialogue/code in this `menu` block have an extra level of indentation. Please remove one level of indentation from **all** the code in this `menu` block.

**Comment by LordMau** [comment]
Reminder that not everyone deleted her to finish the game before downloading this.

File: Monika After Story/game/script-topics.rpy [hunk 13]
```diff
@@ -4827,35 +4827,35 @@ init 5 python:
...
+	m "What are the color of my eyes?"
+	"Green":
+		m "Ahaha, you're right!"
+		m "Well, at least that means you're not color blind."
+		m "Unless you lied to me, but why would you?"
+		m "Ah, in that case you, maybe just didn't want to worry me."
+		m "I'm always willing to deal with any of your worries [player]!"
+	"Gray":
+		m "Gosh, we might have an issue here."
+		m "If you see everything on a scale of greys, that could mean you're completely color blind!”
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: The quote at the end of this line is a bad one. Please replace with regular double quote.

File: Monika After Story/game/script-topics.rpy [hunk 14]
```diff
@@ -4865,45 +4865,44 @@ init 5 python:
...
+     m "Hey [player], have you ever thought about how much time you've spent waiting in lines before?”
+     m "I mean, across your entire life of course!"
+     m "It's probably quite a long time, you know?"
+     m "I bet that time would double if we were to count more than just physical lines!"
+     m "What I mean by that is there's loads of different types of lines!"
+     m "You got those times where you get a number and sit down, waiting for your number to be called!"
+     m "Those might sound rather calm but honestly, there's an even better option!"
+     m "You could just get a place in line in advance with your phone and get a notification when it's your turn!"
+     m "Or you could, you know, spend some time with me while you wait ehehe~"
+     Menu:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: `menu` needs to be lowercase

File: Monika After Story/game/script-topics.rpy [hunk 15]
```diff
@@ -4913,38 +4912,38 @@ init 5 python:
...
-	return
+     m "Have you ever paid attention to the leaves around you [player]?"
+     m "You may have gotten frustated at them falling during autumn."
+     m "This might sound a bit silly to you, but I feel quite bad for those poor leaves."
+     m "Specifically, I mean leaves from deciduous plants."
+     m "These kind of plants lose their leaves to survive harsh weathers, while different sorts of plants do the opposite."
+     m "Those other types of plants are called evergreen and don't need to lose their leaves to survive."
+     m "I know leaves don't have feelings after all, but can you imagine how they must feel?"
+     m "That feeling of love and care sinking into pain and betrayal!"
+     Menu:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: `menu` needs to be lowercase

File: Monika After Story/game/script-topics.rpy [hunk 16]
```diff
@@ -4954,44 +4953,44 @@ init 5 python:
...
-			m "I used to be stuck in a written path and it was horrible."
-			m "No matter what I did I couldn't be with you!"
+     m "Hey [player], have you ever gone to a fortune teller before?"
+     m "You know, paying to have your destiny told?"
+     m "Being able to predict the future is such a weird practice in my opinion."
+     m "Some people will cling to it and make life changing decisions based on it!"
+     m "I find it a bit silly that some readings of cards, hands or similar stuff holds so much importance to people."
+     m "I wonder how much of those who perform such readings, end up getting from that work."
+     m "I suppose it's good to give people some hope, yet when the disappointment appears it usually makes them feel worse than before."
+     Menu:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: `menu` needs to be lowercase

File: Monika After Story/game/script-topics.rpy [hunk 17]
```diff
@@ -5001,39 +5000,39 @@ init 5 python:
...
+     m "Yet right now, AI is super limited."
+     m "Sure it can perform some tasks but surely that can't be qualified as intelligence at the moment!"
+     m "That's just one aspect of it, and really, it is not even that big a part!"
+     m "If AI can be truly called intelligent, then it needs to learn from human psychology."
+     m "There used to be this old test called the turing Test."
+     m "It would put a person and make him or her talk to an AI."
+     m "If the person failed to notice it was an AI then that AI passed the test."
+     m "Yet that's outdated nowadays."
+     m "I’m sure that I look nothing like an AI to you!"
+     Menu:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: `menu` needs to be lowercase

File: Monika After Story/game/script-topics.rpy [hunk 18]
```diff
@@ -4790,6 +4790,298 @@ label monika_urgent:
...
+		m "You can't do anything other than wait for the end like that."
+		m "Why would you deny making your own destiny!"
+		m "I used to be stuck in a written path and it was horrible."
+		m "No matter what I did I couldn't be with you!"
+			m "I am glad that we changed that, [player]."
+     return  
+
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['AI', 'Artificial Intelligence']:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: keys need to be lowercase.

**Comment by Winblows2** [comment]
More syntax errors than cells in the human body. (sarcasm)

**Comment by monikLover** [comment]
Wrong branch. Base PR in 'content.' Closing.

**Pull request closed without merging**